subsys/bluetooth/host/sdp.c

```diff
@@ -2138,6 +2138,10 @@ static ssize_t sdp_get_attr_value_len(const uint8_t *data, size_t len)
 
 	BT_DBG("Attr val DTD 0x%02x", data[0]);
 
+	if (len < 1) {
+		goto err;
+	}
```

Comment:
I'm curious as to why you didn't just log and return the error message here, instead of creating a label and use `goto`.

Reply:
Just using the same flow pattern as the other functions, otherwise I would have done it like you say.